app/views/animals/new.html.erb

```diff
@@ -0,0 +1,18 @@
+
+
+<h1>Add the details of your animal</h1>
+
+<div class="container">
+  <%= simple_form_for(@animal,  html: { class: "form-group w-50 p-5 mx-auto"}) do |f| %>
+    <%= f.input :name %>
+    <%= f.input :species %>
```

Comment:
Set the label to something else than species as we discussed before?

Reply:
done